packages/core/src/components/forms/_control-group.scss

```diff
@@ -229,6 +229,9 @@ Styleguide control-group
 
   Styleguide control-group-fill
   */
+  &.#{$ns}-fill {
```

Comment:
while looking into this bug I found some strange code which I don't think should exist. we have special logic in the control group example to make the "fill" option work on the page... this logic is outside the Blueprint API so it's misleading for developers testing the component. please remove these lines in this PR: https://github.com/palantir/blueprint/blob/842b2bb43f43ad11b71cdec86fff410d1a14474c/packages/docs-app/src/examples/core-examples/controlGroupExample.tsx#L46-L49